cluster-setup/ci-cluster/feature_sriov/20-sriov-testing-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
This should belong to the test setup (once we have it).
Ideally, the tests find the cluster with the sr-iov feature ready to be used and runs its tests, setting up the namespace and launching the pods, verifying what has to be verified.